presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -1784,6 +1784,20 @@ public void testRowNumberSpecialFilters()
                 "WHERE rn = 0");
     }
 
+    @Test
+    public void testMinMaxN()
+    {
+        assertQuery("" +
```

Comment:
why putting an empty string here?

Reply:
I replicated the style from all the other tests in that file. I think it's because if you don't put it, indentation becomes different between 1st and 2nd line
<img width="494" alt="image" src="https://user-images.githubusercontent.com/23248585/199953075-99029e8f-0256-48f7-8ef0-d50dbc094937.png">